Classes/Notifications/Notification+NotificationViewModel.swift

```diff
@@ -8,49 +8,54 @@
 
 import UIKit
 import GitHubAPI
+import FlatCache
 
-func CreateNotificationViewModels(
+func createNotificationViewModels(
     width: CGFloat,
     contentSizeCategory: UIContentSizeCategory,
     v3notifications: [V3Notification],
+    cache: FlatCache,
     completion: @escaping ([NotificationViewModel]) -> Void
     ) {
-    DispatchQueue.global().async {
```

Comment:
Why are we removing these? Wont this execute on the main thread? The models these methods generate include warming text size caches, expensive!

Reply:
I was unsure about this one. The cache need the main queue and I couldn't see an obvious reason why this Dispatch was occurring. 

Will add it back in. 👍